terminus-op/templates/tdb-persistentVolume.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: {{ .Release.Name}}-persistentvolume
+spec:
+  claimRef:
+    name: {{ .Release.Name}}-pvc
+  storageClassName: terminus-op-storage-class

```

Comment:
We should make the storage class a variable instead in my opinion, so that the user can set one.

Reply:
The variable option can be done, will do :)